assume-role

```diff
@@ -0,0 +1,7 @@
+set +x
```

Comment:
Do you need this file if you are building from `awscli` image?

Reply:
Nope.  Removed.